app/login/loginController.js

```diff
@@ -13,6 +13,7 @@
 
             var redirectUser = function(){
                 if(authorizationService.isLoggedIn){
+                    $scope.$parent.isLoggedIn = true;
```

Comment:
I consider, we can do it without $scope.parent, use for example authorizationService